cmd/login/runner.go

```diff
@@ -31,11 +31,21 @@ type runner struct {
 	fs     afero.Fs
 
 	k8sConfigAccess clientcmd.ConfigAccess
+	loginOptions    LoginOptions
 
 	stdout io.Writer
 	stderr io.Writer
 }
 
+type LoginOptions struct {
```

Comment:
This could also be extended with other options. For now I used the ones that were important to the task at hand.